tests/pr_test.go

```diff
@@ -79,3 +81,25 @@ func TestRunUpgradeExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestFSCloudExample(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:      t,
+		TerraformDir: fscloudExampleTerraformDir,
+		Prefix:       "base-ocp-fscloud",
```

Comment:
Do you want to maybe leave a comment here explaining why we don't pass an existing resource group (for example like this: https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/blob/main/tests/pr_test.go#L46-L51